DRAFT blog: life, 2 years after a YC batch (#3252)
PostHog/posthog.com: contents/blog/yc-2-years-on.md

---
date: 2022-05-09
title: YC, 2 years on
rootPage: /blog
sidebar: Blog
showTitle: true
hideAnchor: true
categories: ["CEO diaries", "Inside PostHog"]
author: ["james-hawkins"]
featuredImage: ../images/blog/ceo-diary.png
featuredImageType: full
---

> [YCombinator](https://www.ycombinator.com/) (YC) is seen as the world's best, and most prolific, 3 month accelerator program. Upwards of 7,000 founders have taken part. Yet, no one really talks about what happens afterwards.

Reviewer: ```suggestion
[YCombinator](https://www.ycombinator.com/) (YC) is seen as the world's best, and most prolific, three-month accelerator program. Upwards of 7,000 founders have taken part. Yet, no one really talks about what happens afterwards.
```

> > PostHog did YCombinator's Winter 2020 batch. We made the top companies list a year later when we closed a series B led by YC Continuity. We provide an open source product analytics suite.

Reviewer: ```suggestion
PostHog did YCombinator's Winter 2020 batch. We made the top companies list a year later when we closed a series B led by YC Continuity. We provide an open-source product analytics suite.
```

So, what have the last two (and a bit) years like been for us, and how could they have been better?

## The network

It's a vague term, what does this _actually_ mean?

> The tools we have are an internal forum, dozens of unofficial YC Whatsapp/Slack groups filled with other founders, regular invites to informal in person events, and the contact details of all the partners and every founder. This extends to real life - we met many people there that are now good friends.

Reviewer: ```suggestion
The tools we have are an internal forum, dozens of unofficial YC WhatsApp/Slack groups filled with other founders, regular invites to informal in person events, and the contact details of all the partners and every founder. This extends to real life - we met many people there that are now good friends.
```

We've reduced how often we feel the need to do office hours a lot. During the batch, it was without exception every 2 weeks. Now, it's every couple of months. I think this is because post product market fit, it has felt easier to know what to build - we have many users to talk to and a clear [ideal customer profile](../handbook/strategy/strategy#target-customers-for-2022). This means we usually know what we need to do, and simply doing the work as fast as we can.

However, when we do need help - it's pretty leveraged. For example, we recently confirmed that the majority of our growth comes from word of mouth. We wanted to talk to someone relevant about this - YC meant we could talk to Nicolas Dessaigne (Algolia cofounder) about how he'd approach the problem. This had a major impact on how we think about marketing. As a result, PostHog's marketing team objective is literally "accelerate word of mouth growth" today.

## Money...

> YC has a Continuity Fund. This can invest in later stage companies.

Reviewer: ```suggestion
YC has a Continuity Fund that invests in later stage companies.
```

> We've been fortunate to partner with them. They've:

Reviewer: ```suggestion
We've been fortunate to partner with them. They have:
```

* invested roughly $20M
* helped us stay focused on (the right) users in board meetings
* provided endless further intros to people for advice and hiring as needed

This isn't guaranteed though, but it wouldn't have happened if we'd not gone through the batch.

## ... and how to spend it (hiring)

> [Work at a Startup](https://www.workatastartup.com/) is provided to YC companies for during and after the batch.

Reviewer: ```suggestion
[Work at a Startup](https://www.workatastartup.com/) is provided to YC companies during and after the batch.
```

This has given us a bunch of interviews with candidates.

## Stonks / angel investing side hustle

One of the things that surprised me was how common it is for founders to invest in other founders. Firstly, how come they've all got the cash?! It turns out that some founders act as scouts for VC funds, some have had good tech jobs before, and others have sold some stock. Being part of YC means you have easy access to lots of promising startups.

I did this twice then stopped primarily because I felt it'd get distracting to do it a lot. My personal life [got a bit chaotic](ceo-diary-3) last year.

## Conclusion

There's a lot of ongoing support. A lot of this is provided through network effects so it scales surprisingly well.

> We've both given and received advice from dozens (probably 50 or more) of other companies. This has given us years of experience in weeks, but it can make it tempting to stop thinking for yourself. I've been suggested in a [full team feedback session](../handbook/people/feedback#full-team-feedback-sessions) that I should work from first principles more often, which was helpful advice.

Reviewer: ```suggestion
We've both given and received advice from dozens (probably 50 or more) other companies. This has given us years of experience in weeks, but it can make it tempting to stop thinking for yourself. I've been suggested in a [full team feedback session](../handbook/people/feedback#full-team-feedback-sessions) that I should work from first principles more often, which was helpful advice.
```

> We've both given and received advice from dozens (probably 50 or more) of other companies. This has given us years of experience in weeks, but it can make it tempting to stop thinking for yourself. I've been suggested in a [full team feedback session](../handbook/people/feedback#full-team-feedback-sessions) that I should work from first principles more often, which was helpful advice.

Reviewer: ```suggestion
We've both given and received advice from dozens (probably 50 or more) of other companies. This has given us years of experience in weeks, but it can make it tempting to stop thinking for yourself. It's been suggested in a [full team feedback session](../handbook/people/feedback#full-team-feedback-sessions) that I should work from first principles more often, which was helpful advice.
```

> We've both given and received advice from dozens (probably 50 or more) other companies. This has given us years of experience in weeks, but it can make it tempting to stop thinking for yourself. I've been suggested in a [full team feedback session](../handbook/people/feedback#full-team-feedback-sessions) that I should work from first principles more often, which was helpful advice.

Reviewer: ```suggestion
We've both given and received advice from dozens (probably 50 or more) other companies. This has given us years of experience in weeks, but it can make it tempting to stop thinking for yourself. It's been suggested in a [full team feedback session](../handbook/people/feedback#full-team-feedback-sessions) that I should work from first principles more often, which was helpful advice.
```

> Most importantly, and as cheesy a conclusion as this is, it has changed how we see ourselves. Being part of YC feels means we see people we know very well go on to build very successful companies, sharing what they're up to along the way. That has taken us from feeling like outsiders to feeling like it's very achievable to build something with a lot of impact.

Reviewer: ```suggestion
Most importantly, and as cheesy a conclusion as this is, it has changed how we see ourselves. Being part of YC means we see people we know very well go on to build very successful companies, sharing what they're up to along the way. That has taken us from feeling like outsiders to feeling like it's very achievable to build something with a lot of impact.
```

---

> date: 2022-05-09

Reviewer: ```suggestion
date: 2022-05-18
```

> title: YC, 2 years on

Reviewer: ```suggestion
title: Reflecting on YC, 2 years on
```

rootPage: /blog
sidebar: Blog
showTitle: true
hideAnchor: true
categories: ["CEO diaries", "Inside PostHog"]
author: ["james-hawkins"]

> featuredImage: ../images/blog/ceo-diary.png

Reviewer: ```suggestion
featuredImage: ../images/blog/posthog-ceo-diary-blog.png
```
